### ISSUE TITLE
invalid literal for int() with base 10: 'no'

### ISSUE THREAD

**Issue by joshmoore** [German-BioImaging/dtplatform#23]
```
Failed to convert Literal lexical form to value. Datatype=http://www.w3.org/2001/XMLSchema#int, Converter=<class 'int'>
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/dt/lib/python3.10/site-packages/rdflib/term.py", line 2084, in _castLexicalToPython
    return conv_func(lexical)  # type: ignore[arg-type]
ValueError: invalid literal for int() with base 10: 'no'
```

when running the output of:

`jupyter nbconvert --to python ...`

**Comment by joshmoore** [comment]
Updated steroid use to:

```
    if row["steroid use after V1"]=="yes":
        rdf.add((steroidUse, RDF.value, Literal(True, datatype=XSD.boolean)))
    else:
        rdf.add((steroidUse, RDF.value, Literal(False, datatype=XSD.boolean)))
```

**Comment by joshmoore** [comment]
Also fixed by 1a259bfbc1cc47c0541629a45c0da1eb8ef05c8a
